rust/core-lib/src/client.rs

```diff
@@ -19,7 +19,7 @@ use std::time::Instant;
 use serde_json::{self, Value};
 use xi_rpc::{self, RpcPeer};
 
-use tabs::ViewId;
```

Comment:
Is this from rustfmt or something? Normally we don't include braces on a single-item import.